pkg/parser/model/model.go

```diff
@@ -1687,11 +1687,13 @@ func (fk *FKInfo) Clone() *FKInfo {
 
 // DBInfo provides meta data describing a DB.
 type DBInfo struct {
-	ID                 int64            `json:"id"`      // Database ID
-	Name               CIStr            `json:"db_name"` // DB name.
-	Charset            string           `json:"charset"`
-	Collate            string           `json:"collate"`
-	Tables             []*TableInfo     `json:"-"` // Tables in the DB.
+	ID         int64    `json:"id"`      // Database ID
+	Name       CIStr    `json:"db_name"` // DB name.
+	Charset    string   `json:"charset"`
+	Collate    string   `json:"collate"`
+	Deprecated struct { // Tables is not set in infoschema v2, use infoschema SchemaTableInfos() instead.
+		Tables []*TableInfo `json:"-"` // Tables in the DB.
+	}
```

Comment:
So this is all the core changes of the PR

Reply:
Is there a plan to remove this field?